certification/internal/engine/engine.go

```diff
@@ -437,7 +437,7 @@ func getBgName(srcrpm string) string {
 func writeRPMManifest(ctx context.Context, containerFSPath string) error {
 	pkgList, err := rpm.GetPackageList(ctx, containerFSPath)
 	if err != nil {
-		log.Error("could not get rpm list, continuing without it")
+		log.Error(fmt.Errorf("%w: %s", errors.ErrRPMPackageList, ", continuing without it"))
```

Comment:
Can you  include the value of `err` in your message? You can probably take "continuing without it" and add it straight into your string formatting, as well.

Reply:
Since we continue anyways I think adding the `err` adds unnecessary noise in this case.

Reply:
That's fair, but it still would be easier to read if it's in the format string itself.

Reply:
It does add noise, but we would have nothing to use for debug purposes if we ran into failure cases (where we can't find the RPM list, but are expecting it).

Reply:
As a compromise, I wrapped the error in both checks that call `GetPackageList` but do not think the added noise here adds any value to us when trying to  debug.